### PR TITLE
add logs for stealth errors

### PR DESCRIPTION
**Pull request by Bee-lee**
# Description

Some errors and weird behaviour are not logged properly, but it's happening in prod.

### CODE REVIEW

**Review by epapbak — APPROVED** [review]

**Review by dpensi — APPROVED** [review]

File: server/handlers.go [hunk 1]
```diff
@@ -155,11 +155,12 @@ func (server HTTPServer) overviewEndpoint(writer http.ResponseWriter, request *h
 	for _, clusterID := range clusters {
 		overview, err := server.getOverviewPerCluster(clusterID, authToken, writer)
 		if err != nil {
-			log.Info().Msgf("Problem handling report for cluster %s", clusterID)
+			log.Error().Err(err).Msgf("Problem handling report for cluster %s. Full error message %v.", clusterID, err)
 			continue
 		}
 
 		if overview == nil {
+			log.Error().Msgf("Overview for cluster %v is nil. Skipping.")
```
Comment (tisnik) [inline server/handlers.go:163]: clusterID is missing there

Comment (epapbak) [reply server/handlers.go:163]: true, true

Comment (Bee-lee) [reply server/handlers.go]: blind as a bat, thanks

**Review by tisnik — APPROVED** [review]
LGTM, thanks a lot @Bee-lee

**Pull request merged**